packages/react-components/src/templates/NetworkListPage.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import css from '@emotion/css';
+
+import {
+  perRem,
+  contentSidePaddingWithNavigation,
+  tabletScreen,
+} from '../pixels';
+import { pearl } from '../colors';
+import NetworkPageHeader from './NetworkPageHeader';
+
+const styles = css({
+  alignSelf: 'stretch',
+  background: pearl.rgb,
+  display: 'grid',
+  gridRowGap: `${24 / perRem}em`,
+  padding: `${36 / perRem}em ${contentSidePaddingWithNavigation(8)} ${
+    36 / perRem
+  }em `,
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
```

Comment:
Tip: check out https://github.com/yldio/asap-hub/blob/aecb0704cc060eed981faa3617b5f98b45745e73/packages/react-components/src/atoms/Card.tsx#L14-L20 - we have a `vminLinearCalc` helper that can be used to avoid media queries and instead have things like paddings scale linearly from a small value for mobile to a large value for desktop